Source/Core/VideoBackends/Software/SWVertexLoader.cpp

```diff
@@ -192,6 +194,7 @@ void SWVertexLoader::LoadVertex()
 
 	// transform this vertex so that it can be used for rasterization (outVertex)
 	OutputVertexData* outVertex = m_SetupUnit->GetVertex();
```

Comment:
Might be worth checking if that `memset` is better off being inside `GetVertex` there.